modules/common/src/main/scala/surge/internal/utils/MdcExecutionContext.scala

```diff
@@ -0,0 +1,11 @@
+package surge.internal.utils
+
+import java.util.concurrent.Executors
+import scala.concurrent.ExecutionContext
+
+object MdcExecutionContext {
```

Comment:
I don't think you need this file ;-)

Reply:
Changed it with scala global execution context